test/org/mockitointegration/NoJUnitDependenciesTest.java

```diff
@@ -4,36 +4,75 @@
 import org.junit.Test;
 import org.mockito.Mockito;
 import org.mockito.cglib.proxy.Enhancer;
+import org.mockito.junit.MockitoJUnit;
 import org.mockitoutil.ClassLoaders;
 import org.objenesis.Objenesis;
 
 import java.util.Set;
 
+import static org.fest.assertions.Assertions.assertThat;
+
 public class NoJUnitDependenciesTest {
+
     @Test
-    public void pure_mockito_should_not_depend_JUnit() throws Exception {
+    public void pure_mockito_should_not_depend_JUnit() throws Throwable {
```

Comment:
I like the idea of using isolated class loaders to test classpath related features.